src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs

```diff
@@ -762,7 +762,12 @@ protected sealed override bool Receive(object message)
         private void FailChunkExecution(ChunkExecutionFailure message)
         {
             _remainingOperations++;
-            var cause = message.Cause;
+
+            var cause = message.Cause is AggregateException aggregateException
+                ? aggregateException.Flatten().InnerExceptions.OfType<DbException>().FirstOrDefault()
+                    ?? aggregateException.Flatten().InnerExceptions[0]
+                : message.Cause;
+
```

Comment:
An alternative to this would be to stop throwing an `AggregateException` in line [#1003](https://github.com/Arkatufus/akka.net/blob/4ad16ccfb4f044d164bd13bff2ba99e19abd3b29/src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/BatchingSqlJournal.cs#L1003)